### DISCUSSION TITLE
Is there a way to use Sentry with WasmJs?

### DISCUSSION THREAD

**Discussion by o-nos** [getsentry/sentry-kotlin-multiplatform#243] [Q&A] [answered]
I've recently started digging KMP, and realised I can't use Sentry in the web module. It's alright and fine for Android and iOS modules, but Gradle seems to not find the `sentry-kotlin-multiplatform` lib for the web part of my project.
Could someone help me, please?

The error:

```
:composeApp:wasmJsMain: Could not resolve io.sentry:sentry-kotlin-multiplatform:0.7.1.
Required by:
    project :composeApp
```

**Comment by buenaflor** [comment, marked as answer]
hey, we currently do not support wasm. 

in order to use Sentry you need to separate the sourceSets that we do support.

https://github.com/getsentry/sentry-kotlin-multiplatform/issues/227 please upvote this issue if you want to see support for this

**Reply by o-nos** [reply]
thank you, that's the answer! :)
